src/panel/context/Request.tsx

```diff
@@ -99,6 +101,14 @@ export const RequestProvider: FC = ({ children }) => {
     });
   }, []);
 
+  useEffect(() => {
+    if (!query) {
+      localStorage.removeItem("urql-last-request");
+    } else {
+      localStorage.setItem("urql-last-request", query);
+    }
```

Comment:
Non blocking, I think we should keep the state identical between reloads, even if it's empty

```suggestion
    localStorage.setItem("urql-last-request", query);
```